scripts/poChecker.py

```diff
@@ -104,33 +114,60 @@ def _checkMessages(self):
 			if line.startswith(self.FUZZY):
 				nextFuzzy = True
 				continue
-			elif line.startswith(self.MSGID):
+			elif line.startswith(self.MSGID) and not line.startswith(self.MSGID_PLURAL):
 				# New message.
-				self._msgstr = self._finishString()
-				if self._msgstr:
+				if self._msgstr is not None:
+					self._msgstr[-1] = self._finishString()
 					# Check the message we just handled.
 					self._checkMessage()
 				command = self.MSGID
 				start = command
 				self._messageLineNum = lineNum
 				self._fuzzy = nextFuzzy
 				nextFuzzy = False
-			elif line.startswith(self.MSGSTR):
+			elif line.startswith(self.MSGID_PLURAL):
 				self._msgid = self._finishString()
+				command = self.MSGID_PLURAL
+				start = command				
```

Comment:
should this be moved to before `elif line.startswith(self.MSGID) and not line.startswith(self.MSGID_PLURAL)`? that would mean that checking the second condition would be unnecessary

Reply:
When I implemented this PR, I had first coded something similar as your proposal.

But I have changed things and I felt that things are clearer and easier to follow in my current proposal:
* Each case appears in the order they appear in the .po file
* Each case  is independant, i.e. all the branches of the if/elif could be reordered without any impact.
* the overhead of evaluating twice the same condition is probably not significant

Given this information, let me know if you still want me to switch the two branches of the if/elif statements or not. Thanks.